dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/EntityRepositoryImpl.java

```diff
@@ -32,14 +32,19 @@
   public static final String TABLE_NAME = "entities";
 
   public static String getSqlInsertFields() {
-    return IdentifiableRepositoryImpl.getSqlInsertFields() + ", custom_attrs, entity_type";
+    return IdentifiableRepositoryImpl.getSqlInsertFields()
+        + ", custom_attrs"
+        + ", entity_type"
+        + ", navdate";
```

Comment:
Das `navdate` müsste in `getColumnName` auch noch ergänzt werden, sonst ist `order by navdate` nicht möglich.

Reply:
Ist ergänzt :+1: